client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/index.tsx

```diff
@@ -10,20 +11,15 @@ import FormattedHeader from 'calypso/components/formatted-header';
 import { LoadingEllipsis } from 'calypso/components/loading-ellipsis';
 import WarningCard from 'calypso/components/warning-card';
 import { useSite } from 'calypso/landing/stepper/hooks/use-site';
-import {
-	AUTOMATED_ELIGIBILITY_STORE,
-	SITE_STORE,
-	ONBOARD_STORE,
-	PRODUCTS_LIST_STORE,
-} from 'calypso/landing/stepper/stores';
+import { SITE_STORE, ONBOARD_STORE, PRODUCTS_LIST_STORE } from 'calypso/landing/stepper/stores';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { addQueryArgs } from 'calypso/lib/url';
 import { ActionSection, StyledNextButton } from 'calypso/signup/steps/woocommerce-install';
 import { eligibilityHolds as eligibilityHoldsConstants } from 'calypso/state/automated-transfer/constants';
 import SupportCard from '../store-address/support-card';
 import type { Step } from '../../types';
 import type { ProductsListSelect, OnboardSelect, SiteSelect } from '@automattic/data-stores';
-import type { TransferEligibilityError } from '@automattic/data-stores/src/automated-transfer-eligibility/types';
+import type { TransferEligibilityHold } from '@automattic/data-stores/src/automated-transfer-eligibility/types';
```

Comment:
This is definitely odd. I wonder if we can export the type as part of the `@automattic/data-stores` package and import it from there as well, without the extra sub-path?

Reply:
Turns out we didn't even need it since type inference worked where it was used